include/openssl/ct.h

```diff
@@ -481,7 +485,7 @@ int i2o_SCT_signature(const SCT *sct, unsigned char **out);
 * |len| should be the length of the signature in |in|.
 * Returns the number of bytes parsed, or a negative integer if an error occurs.
 */
-int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
+__owur int o2i_SCT_signature(SCT *sct, const unsigned char **in, size_t len);
```

Comment:
Since you'll hardly ever have a signature without an accompanying SCT, a method that parses the signature straight into the SCT seems fine for now. How about `SCT_set1_serialized_signature`? It should not be `o2i`, and it should distinguish itself from `SCT_set1_signature` which only sets the signature, not the hash and sig algo.

Note that the fact that SCT stores the sig as separate fields is an implementation detail. So `SCT_set1_serialized_signature` seems appropriate.